src/core/error/common_error.rs

```diff
@@ -451,6 +451,12 @@ pub enum CommonError {
 
     #[error("Invalid LedgerHardwareWalletModel, bad value: {bad_value}")]
     InvalidLedgerHardwareWalletModel { bad_value: String } = 10126,
+
+    #[error("RadixConnectMobile invalid URL")]
+    RadixConnectMobileInvalidRequestUrl = 10127,
+
```

Comment:
copy paste typo in description, and also you might wanna include a `{ bad_value: String }`, see `InvalidLedgerHardwareWalletModel` above

Reply:
ofc making use of `bad_value` in description as seen in:

```rust
  #[error("Invalid LedgerHardwareWalletModel, bad value: {bad_value}")]
```